themes/navy/layout/index.swig

```diff
@@ -44,10 +44,18 @@
 
     <div class="command-wrap">
         <div class="command">
-            npm install -g embark
+            <span id="npm-current">npm install -g embark</span>
+            <button class="btn" data-clipboard-target="#npm-current"><img class="clippy" width="13" src="/img/clippy.svg" alt="Copy to clipboard"></button>
         </div>
         <div class="command-description">
-            Install Embark via NPM
+            Install Embark via <a href="https://github.com/creationix/nvm" target="_blank">NPM</a>
+        </div>
+        <div class="command-description">
+            To get access to Embark 4 alpha:
```

Comment:
Same here `next` won't point at 4.0.0-alpha.x in the future anymore. We have to keep this timeless.